components/brave_wallet_ui/components/desktop/views/accounts/account.tsx

```diff
@@ -52,6 +52,8 @@ import { useBalance } from '../../../../common/hooks/balance'
 
 // Actions
 import { WalletPageActions } from '../../../../page/actions'
+import { CoinType } from 'gen/brave/components/brave_wallet/common/brave_wallet.mojom.m'
```

Comment:
Since we already have the `BraveWallet` type imported in this file, let's remove this import and use `BraveWallet.CoinType.FIL` 😎